### PR TITLE
[#168779317] add User Identity get API

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#168779317](https://www.pivotaltracker.com/story/show/168779317): Creare un nuovo endpoint che restituisce SOLO i dati ottenuti tramite il processo di autenticazione

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api_backend.yaml [hunk 1]
```diff
@@ -307,6 +307,35 @@ paths:
           description: Profile cannot be updated.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/user-identity":
+    x-swagger-router-controller: ProfileController
+    get:
+      operationId: getSpidProfile
```
Comment (francescopersico) [inline api_backend.yaml]: We are going to add new authentication methods so i think is better to be generic (no hardcoded SPID).

@cc @cloudify

Comment (cloudify) [reply api_backend.yaml]: agreed

Comment (BurnedMarshal) [reply api_backend.yaml]: A more generic `getUserIdentity` could be better?

Comment (BurnedMarshal) [reply api_backend.yaml]: Fixed in 2e6a1bf

Comment (gunzip) [reply api_backend.yaml]: it looks like it's not commited yet

Comment (BurnedMarshal) [reply api_backend.yaml]: Fixed in 4fd9645

File: api_backend.yaml [hunk 2]
```diff
@@ -329,9 +329,9 @@ paths:
               version: 1
               sender_allowed: true
         "400":
-          description: Bad request
-          schema:
-            $ref: "#/definitions/ProblemJson"
+            description: Bad request
```
Comment (gunzip) [inline api_backend.yaml]: why this change ? it looks like the former revision was right
